gato/gatos/BladeGato.py

```diff
@@ -0,0 +1,59 @@
+from ABaseGato import ABaseGato, require_alive
+
+
+class BladeGato(ABaseGato):
+    """
+        > Fav Food: Spider Lily
+        > Every time HP decreases, increases all other stats by 20% the decreased amount. Every time HP increases, decreases all other stats by 20% the increased amount.
+        > Eidolons: e1->e6 increases by 3 each. (20% at E0, 38% at E6)
+        > E6: increase max hp by 20
+    """
+
+    # TODO: img and animation
+    IMAGE: str = "https://cdn.discordapp.com/attachments/1198010808672723075/1201435790845165658/nuxiom_gato.PNG"
+    ANIMATIONS: str = "bladegato"
```

Comment:
at least one of image, animation and display names are missing for each gato atm